src/directives/stream.js

```diff
@@ -45,7 +45,8 @@ export default {
         })
       })
     } else {
-      if (!Rx.Observable.fromEvent) {
+      const fromEvent = Rx.fromEvent || Rx.Observable.fromEvent
```

Comment:
This is apparently using the umd global?

All locations in the v6 global are a mirror of the module locations, only with periods. This should be:

`const fromEvent = rxjs.fromEvent`

Reply:
This library passes in it's own `Rx` that comes from the configuration, so `Rx` is valid here.